stories/questions.stories.js

```diff
@@ -0,0 +1,32 @@
+// https://github.com/EnCiv/undebate-ssp/issues/48
+
+import React, { useState } from 'react'
+
+import Questions from '../app/components/questions'
+
+export default {
+    title: 'Questions',
+    component: Questions,
+    argTypes: {},
+}
+
+const Template = args => {
+    const [validity, setValidity] = useState({ valid: false, value: '' })
+    return (
+        <div>
+            <Questions {...args} onDone={v => setValidity(v)} />
+            <p>Is valid: {validity.valid ? 'True' : 'False'}</p>
+            <p>Value: {validity.value}</p>
+        </div>
+    )
+}
+
+export const QuestionsTest = Template.bind({})
+QuestionsTest.args = {
+    electionOM: {
```

Comment:
see the comments above the electionOM in an array - not an obj.  Also see comments about upsert.